docs/conf.py

```diff
@@ -188,9 +188,7 @@
         "json_url": json_url,
         "version_match": version_match,
     },
-    "navigation_with_keys": False,
     # "back_to_top_button": False,
-    # "search_bar_position": "navbar",  # TODO: Deprecated - remove in future version
```

Comment:
AFAICT this config was being silently ignored, and associated code was already removed